pg_views/management/commands/show_view_to_model_diff.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+from django.apps import apps
+from django.conf import settings
+from pg_views.loading import get_sql_model_views
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
```

Comment:
testoval jsi to nebo je to jen můj okopčený kód? :D

Reply:
já ho psal z hlavy nezkoušel jsem ho

Reply:
jo, ten tvuj nefungoval, trochu jsem to predelal

Reply:
cool